extension/chrome/elements/pgp_block_modules/pgp-block-signature-module.ts

```diff
@@ -41,41 +51,49 @@ export class PgpBlockViewSignatureModule {
         $('#pgp_signature > .result').text('signature does not match');
         this.view.renderModule.setFrameColor('red');
       }
-    } else {
-      $('#pgp_signature').addClass('bad');
-      $('#pgp_signature > .cursive').remove();
-      $('#pgp_signature > .result').text('Message Not Signed');
+      if (signature) {
+        PgpBlockViewSignatureModule.setSigner(signature);
+      }
+      this.view.renderModule.doNotSetStateAsReadyYet = false;
+      Ui.setTestState('ready');
     }
-    $('#pgp_signature').css('block');
+    // $('#pgp_signature').css('block');
```

Comment:
I don't really understand what this line was for. Can we remove it?

Reply:
Looks to be a buggy line. It should say `.css('display', 'block')` but if it worked anyway, I'd remove it as you say.